src/hotspot/cpu/arm/c1_MacroAssembler_arm.cpp

```diff
@@ -212,38 +214,51 @@ int C1_MacroAssembler::lock_object(Register hdr, Register obj, Register disp_hdr
 
   assert(oopDesc::mark_offset_in_bytes() == 0, "Required by atomic instructions");
 
-  // On MP platforms the next load could return a 'stale' value if the memory location has been modified by another thread.
-  // That would be acceptable as ether CAS or slow case path is taken in that case.
+  if (LockingMode == LM_LIGHTWEIGHT) {
+    log_trace(fastlock)("C1_MacroAssembler::lock fast");
```

Comment:
Here and later: I don't think we need these log statements?

Reply:
They had been helpful during development, to verify code generation. Going forward they can be scrapped, but lets do that in a follow-up PR.